components/developer/pkgconf/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Vincent Torri
+#
+BUILD_BITS=64
+BUILD_STYLE=configure
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           pkgconf
+COMPONENT_VERSION=        1.9.3
+COMPONENT_SUMMARY=        Package compiler and linker metadata toolkit
+COMPONENT_PROJECT_URL=    http://pkgconf.org/
+COMPONENT_SRC=            $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=        $(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_URL=    https://distfiles.dereferenced.org/pkgconf/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=   sha256:5fb355b487d54fb6d341e4f18d4e2f7e813a6622cf03a9e87affa6a40565699d
+COMPONENT_FMRI=           developer/build/pkgconf
+COMPONENT_CLASSIFICATION= Development/System
+COMPONENT_LICENSE=        ISC
+
```

Comment:
We are missing the kyua test framework. Thus, you should disable tests here by adding:
TEST_TARGET = $(NO_TESTS)